### DISCUSSION TITLE
Question about a link on https://carpentries-incubator.github.io/bioc-project/

### DISCUSSION THREAD

**Discussion by doujouDC** [carpentries-incubator/bioc-project#67] [General]
Hey All,

I cannot remember if I mentioned this before. This sentence near the bottom of the page: "If you are teaching this lesson in a workshop, please see the [Instructor notes](https://carpentries-incubator.github.io/bioc-project/guide/)." opens to a weird page (see image below). BTW, I will trying this lesson this summer as part of a larger class. 

<img width="983" alt="Screenshot 2023-04-11 at 18 45 08" src="https://user-images.githubusercontent.com/6465940/231304859-b2e3b931-3edc-4518-b33b-dd67a764350d.png">

Cheers

Doug

**Comment by kevinrue** [comment]
Thanks for bringing this up. I haven't worked on the lesson in a while, so it's a wild guess, but I wonder whether it is related to the recent rework into the Carpentries workbench format (https://github.com/carpentries-incubator/bioc-project/commits/main) or whether the issue existed before the rework.

**Comment by mblue9** [comment]
@kevinrue I think the bit with that link can be removed (I'll submit a PR). As it's not in the other bioc- or workbench lessons that I can see. And the "Instructor Notes" in the menu along the top of the episode pages works fine https://carpentries-incubator.github.io/bioc-project/instructor/instructor-notes.html
